Libs/Widgets/ctkDirectoryButton.h

```diff
@@ -31,10 +31,6 @@
 #include "ctkWidgetsExport.h"
 class ctkDirectoryButtonPrivate;
 
-// QFileDialog::Options can be used since Qt 4.7.0 (QT_VERSION >= 0x040700)
-// it is disabled to support older Qt versions
-//#define USE_QFILEDIALOG_OPTIONS 1
```

Comment:
I suggest to perform this update in a follow-up pull request as it does not strictly remove code.

Reply:
I've pushed a separate commit as `ENH: Remove custom CTK Dialog options`